client/jetpack-cloud/sections/partner-portal/license-bundle-card/style.scss

```diff
@@ -7,6 +7,11 @@
 	justify-content: space-between;
 	box-sizing: border-box;
 
+	&--with-background {
+		background-image: url(./gradient.svg);
+		background-size: 100% 100%;
+	}
+
```

Comment:
Could we do `license-bundle-card--with-background` instead of `&--with-background`? Searching for the class name will be hard and might get removed without knowing.